pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -53,6 +54,27 @@ type Registry struct {
 	Prefix string `toml:"prefix"`
 }
 
+// IndexInfo holds information of registries in the form of "host:port"
+// and stores whether it is secure or not
+type IndexInfo struct {
```

Comment:
There already is a `Registry`.

BTW the “index” name is a bit confusing — yes, I know Docker uses it.

Reply:
Not sure what to name it, renamed to HostPortConfig for now.

Reply:
I don’t think this needs to exist at all; host[:port][/suffix] entries can already be represented via `tomlConfig.Registries` / `SysRegistries.Registries`.

Non-CIDR values in `tomlConfig.CIDRRanges` should be treated as as syntax errors rather than recorded in a separate `SysRegistries.HostPortConfigs`.